project/profiles/capacity4more/modules/c4m/og/c4m_og_purl/c4m_og_purl.module

```diff
@@ -49,6 +49,11 @@ function c4m_og_purl_url_outbound_alter(&$path, &$options, $original_path) {
     return;
   }
 
+  if (strpos($_GET['q'], 'admin/') === 0) {
```

Comment:
@soniCaH 
This is a bug that we caught during the features export,
can you open a ticket on JIRA for that so I can move this fix (since it is not related to this issue).